spec/features/idv/doc_auth/verify_step_spec.rb

```diff
@@ -20,15 +20,10 @@
     expect(page).to have_content(t('doc_auth.headings.verify'))
   end
 
-  it 'can toggle viewing the ssn' do
-    input_with_ssn_value = "input[value='666-66-1234']"
-    expect(page).to have_selector(input_with_ssn_value), visible: false
-
-    find('input.ssn-toggle').click
-    expect(page).to have_selector(input_with_ssn_value), visible: true
```

Comment:
I'll double-check on Monday, but I have a feeling this test wasn't working the way we expected it to previously, since this is run without JS, and even the previous implementation had relied on JS for the toggling.

Reply:
- `visible` is a parameter to `have_selector`, so it should have been `have_selector(input_with_ssn_value, visible: true)`
- The element is always visible, so `visible: false` vs. `visible: true` makes no difference (`visible: false` ["finds invisible and visible elements"](https://rubydoc.info/github/jnicklas/capybara/Capybara/Node/Finders#all-instance_method)).
- More accurate would have been testing that the `type` changes to show / hide the value, e.g. `expect(page).to have_selector("#{input_with_ssn_value}[type=text]")`. This would have failed though, since JavaScript isn't enabled here.